knowledge-base/chart-customize-separate-markers.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Configuring Individual Markers in a ScatterLine Chart
+description: Learn how to customize markers for specific data points in a ScatterLine Chart to highlight them as special.
+type: how-to
+page_title: How to Customize Markers for Specific Data Points in ScatterLine Chart
+slug: chart-kb-customize-separate-markers
+tags: charts, blazor, scatterline, markers, customization
+res_type: kb
+ticketid: 1666618
+---
+
+## Environment
+
+<table>
+<tbody>
+<tr>
+<td>Product</td>
+<td>Charts for Blazor</td>
+</tr>
+</tbody>
+</table>
+
+## Description
+
+I need to configure a ScatterLine Chart so I can control the marker used for each data point independently. I want to mark some points as "special". 
+
+This KB article answers the following questions:
+- How to mark specific data points in a ScatterLine Chart as special?
+- How to customize separate markers in a Blazor ScatterLine Chart?
+- How to use different markers for special data points in a ScatterLine Chart?
+
+## Solution
+
+The marker type is defined per series level, so it is not possible to set different markers for the different data points out of the box.
+
+To achieve the desired result of marking specific data points as "special", follow these steps:
```

Comment:
It's not very likely for developers to google by "desired result". If you value the "achieve" word, you can place it somewhere else in a simpler structure.

```suggestion
To mark specific data points as "special", follow these steps:
```